src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/step/forcemerge/AttemptCallForceMergeStep.kt

```diff
@@ -87,5 +122,9 @@ class AttemptCallForceMergeStep(
 
     companion object {
         const val name = "attempt_call_force_merge"
+        const val FIVE_MINUTES_IN_MILLIS = 1000 * 60 * 5 // how long to wait for the force merge request before moving on
+        const val FIVE_SECONDS_IN_MILLIS = 1000L * 5L // delay
+        fun getFailedMessage(index: String) = "Failed to start force merge [index=$index]"
+        fun getSuccessMessage(index: String) = "Successfully completed force merge [index=$index]"
```

Comment:
This message can be misleading since this step only calls force merge. `"Successfully completed force merge"` implies the task has completed.

Reply:
It's half right, if there was a response then it did actually complete before the 5 minute timer. But, if the response is still null it just means we passed 5 minutes and it hasn't completed yet. Updated with another message and a conditional.